src/Dfc.ProviderPortal.TribalExporter/Functions/VenueMigrator.cs

```diff
@@ -206,6 +206,8 @@ UNION ALL
                             // Close the SqlDataReader.
                             dataReader.Close();
                         }
+
+                        sqlConnection.Close();
```

Comment:
There isn't a need to add this, as it's in a using statement will dispose of the connection.